tests/qos/test_tunnel_qos_remap.py

```diff
@@ -36,7 +36,7 @@
 
 pytestmark = [
     pytest.mark.enable_active_active,
-    pytest.mark.topology('t0')
+    pytest.mark.topology('dualtor')
```

Comment:
Why make this change? Is this test dualtor specific?

Reply:
Yes, it's dualtor specific test, the mark was wrong